site2/website/release-notes.md

```diff
@@ -1,6 +1,439 @@
+- [issue-12272] PulsarAuthorizationProvider initialization. [12273](https://github.com/apache/pulsar/pull/12273)
+- fix ttl expiry does not take effect. [12266](https://github.com/apache/pulsar/pull/12266)
+- The loadbalancer should avoid offload the heartbeat namespace. [12252](https://github.com/apache/pulsar/pull/12252)
+- fix lost message issues 12221. [12223](https://github.com/apache/pulsar/pull/12223)
+- Disable stats recorder for built-in PulsarClient. [12217](https://github.com/apache/pulsar/pull/12217)
+- [pulsar-server] support v1 api to create non-partition topic. [12122](https://github.com/apache/pulsar/pull/12122)
+- [pulsar-broker] support remove-ttl api for V1 namespace. [12121](https://github.com/apache/pulsar/pull/12121)
+- [Issue 1240][C++] Support setting ConsumerEventListener in pulsar cpp client. [12118](https://github.com/apache/pulsar/pull/12118)
+- [pulsar-broker] Broker auto refresh bk-client certs to avoid cnx failure after cert refresh. [12107](https://github.com/apache/pulsar/pull/12107)
+- [pulsar-client] Fix NPE in pulsar bolt while publishing messages. [12106](https://github.com/apache/pulsar/pull/12106)
+- Fix wrong key-hash selector used for new consumers after all the previous consumers disconnected. [12035](https://github.com/apache/pulsar/pull/12035)
+- [PIP-92] Part-1: Separate Topic Policies into a separate class. [12034](https://github.com/apache/pulsar/pull/12034)
+- Avoid adding duplicated BrokerEntryMetadata. [12018](https://github.com/apache/pulsar/pull/12018)
+- Remove deprecated netty allocator tiny size. [12012](https://github.com/apache/pulsar/pull/12012)
+- [Standalone] Fix the assignment of a custom stream-storage-port number. [11995](https://github.com/apache/pulsar/pull/11995)
+- [testclient] Printing aggregated data when client exit. [11985](https://github.com/apache/pulsar/pull/11985)
+- [standalone] remove noisy log on pulsar standalone startup. [11970](https://github.com/apache/pulsar/pull/11970)
+- Print position info when can't find next valid position.. [11969](https://github.com/apache/pulsar/pull/11969)
+- [Issue #11799][logging] PIP-89: Timed log events. [11944](https://github.com/apache/pulsar/pull/11944)
+- [Issue 11936] forget to call SendCallback on producer close. [11939](https://github.com/apache/pulsar/pull/11939)
+- [website][upgrade]feat: pulsar functions docs migratin for version next/2.8.0/2.7.3. [11938](https://github.com/apache/pulsar/pull/11938)
+- Avoid to infinitely split bundle. [11937](https://github.com/apache/pulsar/pull/11937)
+- Update the default log pattern. [11935](https://github.com/apache/pulsar/pull/11935)
+- [Transaction] add method to clear up transaction buffer snapshot. [11934](https://github.com/apache/pulsar/pull/11934)
+- [pulsar-client]Add a optional params scope for pulsar oauth2 client. [11931](https://github.com/apache/pulsar/pull/11931)
```

Comment:
Nit: this was an update to the Java client.

Reply:
yes. it makes sense to note it in the section above, thanks